### DISCUSSION TITLE
Rules under Detection missing; Alert data missing.

### DISCUSSION THREAD

**Discussion by blindcape** [Security-Onion-Solutions/securityonion#13704] [2.4]
### Version

2.4.100

### Installation Method

Other (please provide detail below)

### Description

configuration

### Installation Type

Standalone

### Location

cloud

### Hardware Specs

Exceeds minimum requirements

### CPU

4

### RAM

32

### Storage for /

256

### Storage for /nsm

256

### Network Traffic Collection

span port

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

Result of: sudo salt-call state.highstate
==========================
local:
    Data failed to compile:
==========================

I have a brand new installation on AWS Cloud.

This is a standalone installation. However, I observed that Rule sets under Detection did not appear at the bottom of the page, despite setting the correct queries.

Similarly, the Alerts under Alert section disappear as well.

These disappearance could not be replicated. It happened seemingly at random, and I could not trigger the conditions that would have made it happened. 

I have did fresh installation 3 times.

Any advice is appreciated. 

Thank you.

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by dougburks** [comment]
> I have a brand new installation on AWS Cloud.

Did you follow the instructions here?
https://docs.securityonion.net/en/2.4/cloud-amazon.html

> This is a standalone installation. However, I observed that Rule sets under Detection did not appear at the bottom of the page, despite setting the correct queries.

Are you able to share any screenshots of this?

Were there any errors during setup?

Have you checked the sosetup log in /root/ for any additional clues?

> Similarly, the Alerts under Alert section disappear as well.

Are you able to share any screenshots of this?

Does data appear in Dashboards or Hunt?
